test-cases/tests/upgrade/n10-verify-quota-feature-upgrade.md

```diff
@@ -112,3 +114,13 @@ echo rate limit resources: $ratelimit_resources
 
 echo requests per limit value: $ratelimit_value
 ```
+
+10. Clone [delorean](https://github.com/integr8ly/delorean) repo and run the following command to generate a downtime report using the delorean cli:
+
+```
+cd delorean
+make build/cli
+./delorean pipeline query-report --namespace redhat-rhoam-observability --config-file ./configurations/downtime-report-config-rhoam.yaml -o <output_dir>
+```
+
+There will be a yaml file generated in the output directory.
```

Comment:
should there be an instruction about what to do with this yaml file? (upload it somewhere, add it to a jira ticket...)

Reply:
As per discussion we agreed that DeLorean report is overkill for Quota check and the workload-web-app would suffice. I removed this part altogether.